test/e2e/apps/disruption.go

```diff
@@ -402,3 +403,18 @@ func waitForPdbToBeProcessed(cs kubernetes.Interface, ns string) {
 	})
 	framework.ExpectNoError(err, "Waiting for the pdb to be processed in namespace %s", ns)
 }
+
+func waitForPdbToObserveHealthyPods(cs kubernetes.Interface, ns string, healthyCount int32) {
+	ginkgo.By("Waiting for the pdb to observed all healthy pods")
+	err := wait.PollImmediate(framework.Poll, schedulingTimeout, func() (bool, error) {
```

Comment:
schedulingTimeout (10 minutes) is too long for this... for a simple observation timeout, wait.ForeverTestTimeout should be fine

Reply:
I have updated the PR.